src/sx-tailwind-website/package.json

```diff
@@ -16,16 +16,15 @@
     "react-syntax-highlighter": "^15.2.1"
   },
   "devDependencies": {
-    "@adeira/babel-plugin-transform-sx-tailwind": "^0.2.0",
+    "@adeira/babel-plugin-transform-sx-tailwind": "^0.3.0",
     "@adeira/babel-preset-adeira": "^1.1.0",
-    "flow-bin": "^0.136.0",
-    "serve": "^11.3.2"
+    "flow-bin": "^0.136.0"
   },
   "scripts": {
     "dev": "next",
     "build": "next build",
     "preexport": "yarn run build",
     "export": "next export",
-    "start": "serve out"
+    "start": "next start"
```

Comment:
Do you still need the `export` and `preexport`? I checked the Vercel configuration and maybe we could change the preset back from `Other` to default `Next.js`?

Reply:
Yep, I'll try to switch to default Nextjs settings.